CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,135 @@
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
```

Comment:
This badge should also be updated with the full URL, i.e. [https://www.contributor-covenant.org/version/2/1/code_of_conduct.html](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html), feel free to get this in along #9

Reply:
Oh didn't see this review but I have put up another PR for that. #11

Reply:
Sorry, I didn't see that either and merged it!

Reply:
No worries, we weren't able to catch that as we don't have link checker configured on this one yet, may be we can look forward to configure it